source/docs/software/frc-glossary.rst

```diff
@@ -0,0 +1,73 @@
+.. include:: <isonum.txt>
+
+FRC Glossary
+============
+
+.. glossary::
+
+   accelerometer
+      A common sensors used to measure acceleration in one or more axis.
+
+   alliance
+      A cooperative of up to four (4) FIRST\ |reg| Robotics Competition teams.
+
+   auto
+      The first phase of each :term:`match` is called Autonomous (auto) and consists of the first fifteen (0:15) seconds.
+
+   COTS
+      Commercial off the shelf, a standard (i.e. not custom order) part commonly available from a vendor to all teams for purchase.
+
+   C++
+      One of the three officially supported programming languages.
+
+   depricated
+      Software that will be maintained for at least 1 year, but may be removed after that.
+
+   DHCP
+      Dynamic Host Configuration Protocol, the protocol that allows a central device to assign unique IP addressess to all other devices.
+
+   FMS
+      Field Management System, the electronics core responsible for sensing and controlling the FIRST Robotics Competition field.
+
+   GradleRIO
+      The mechanism that powers the deployment of robot code to the roboRIO.
+
+   gyroscope
+      A device that measures rate-of-rotation.  It can add up the rotation measurements to determine :term:`heading` of the robot. (“gyro”, for short)
+
+   heading
+      The direction the robot is pointed, usually expressed as an angle in degrees.
+
+   IMU
+      Inertial Measurement Unit, a sensor that combines both an :term:`accelerometer` and a :term:`gyroscope` into a single sensor.
+
+   Java
+      One of the three officially supported programming languages.
+
+   KOP
+      Kit of Parts, the collection of items listed on the Kickoff Kit checklists, distributed to the team via FIRST Choice, or paid for completely (except shipping) with a Product Donation Voucher (PDV).
+
+   KOP chassis
+      The `AM14U4 <https://www.andymark.com/products/am14u4-kit-of-parts-chassis>`__ chassis distributed to every team in the :term:`KOP`.
+
+   LabVIEW
+      One of the three officially supported programming languages.
+
+   match
+      A two (2) minute and thirty (30) second period of time in which an :term:`alliance` plays a FRC game.
+
+   NT
```

Comment:
> NT
Not really anywhere this is referenced as NT. Just leave it as NetworkTables

Reply:
I don't think it is as rare as you think, from our docs:
Twice [here](https://docs.wpilib.org/en/latest/docs/software/vision-processing/roborio/using-multiple-cameras.html?highlight=NT#switching-the-driver-views)
Once [here](https://docs.wpilib.org/en/latest/docs/software/wpilib-tools/shuffleboard/custom-widgets/builtin-plugins.html?highlight=NT#networktables-plugin)
The rPi abbreviates is this way in the logs [here](https://docs.wpilib.org/en/latest/docs/software/vision-processing/wpilibpi/the-raspberry-pi-frc-console.html?highlight=NT#vision-status)
[Here](https://docs.wpilib.org/en/latest/docs/software/networktables/multiple-instances.html?highlight=NT#creating-multiple-instances-of-networktables)

There are probably more.

Reply:
I think in this specific case we should prefer NetworkTables, we can still reference to it when doing NT by doing ```:term:`NT <NetworkTables>` ``` 

We should probably avoid two letter abbreviations as they can quickly collide.